core/src/main/java/io/syndesis/core/Tokens.java

```diff
@@ -38,13 +49,14 @@ public static String getAuthenticationToken() {
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
         if (authentication == null) {
             throw new IllegalStateException("Cannot set authorization header because there is no authenticated principal");
-        } else if (!KeycloakAuthenticationToken.class.isAssignableFrom(authentication.getClass())) {
+        }
```

Comment:
Yeah I'd prefer to do that in a separate PR, it requires a bit of thought (oh no!) because this method is actually called in the git workflow, which doesn't know if it's getting the token from the session or thread local.